contracts/splitter/.cargo/config

```diff
@@ -0,0 +1,2 @@
+[alias]
+schema = "run --bin token-distributor-schema"
```

Comment:
Maybe we should rename it to `drop-splitter-schema` just to avoid confusion? I am saying because there is already a distribution contract which has `drop-distribution-schema`.